Os/test/ut/file/MyRules.cpp

```diff
@@ -0,0 +1,1150 @@
+// ======================================================================
```

Comment:
Why is this file called MyRules.cpp? It doesn't seem like a good name for production code developed by a team. (To whom does "my" refer to in this context?) Maybe just Rules.cpp?

Reply:
FileRules as it is rules for the file test!